### DISCUSSION TITLE
Share your Referer Mod settings recommendations!

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [airtower-luna/referer-mod#43] [Show and tell]
I thought I would create a discussion here so people can share which `referer` modification settings they find beneficial.

Since a default `referer` of `Target` seems to work well in almost all cases, let's assume that default, and share exceptions.

Here's the proposed format:
`Target          | Action | Replacement    | Site Fixed                     | Functionality Fixed            | Discovered`

The date discovered is useful, as the only thing constant is change, but can be a loose approximate if you don't remember.

I'll lead with a few:

`cdpn.io         | Prune  | N/A            | Sites with codepen.io examples | Allow codepen examples to load | 2020/01/01`
`signin.ebay.com | Prune  | N/A            | ebay.com                       | Logging in                     | 2021/10/01`
`hcaptcha.com    | Prune  | N/A            | Don't remember                 | Allow captcha to work          | 2021/01/01`

**Comment by airtower-luna** [comment]
When logging in to GitLab, it uses the Referer to redirect you back to the page where you clicked the login button after. If you want that to work, set the policy for the instance to "keep".
